### PR TITLE
Framework twig

### PR DESCRIPTION
**Pull request by ayumihamsaki**
As per here: https://github.com/octobercms/october/pull/4285 and https://github.com/octobercms/october/issues/4277

### CODE REVIEW

**Comment by LukeTowers** [comment]
@ayumihamsaki I don't understand why all the changes aren't showing up in the files changed tab...

File: modules/cms/twig/FrameworkNode.php [hunk 1]
```diff
@@ -25,13 +25,20 @@ public function __construct($name, $lineno, $tag = 'framework')
...
     {
         $attrib = $this->getAttribute('name');
-        $includeExtras = strtolower(trim($attrib)) == 'extras';
 
         $compiler
             ->addDebugInfo($this)
             ->write("\$_minify = ".CombineAssets::class."::instance()->useMinify;" . PHP_EOL);
 
-        if ($includeExtras) {
+        if (strtolower(trim($attrib)) === 'jquery') {
```
Comment (LukeTowers) [inline modules/cms/twig/FrameworkNode.php]: This is should be handled under the `{% jquery %}` tag, not the framework tag

Comment (ayumihamsaki) [reply modules/cms/twig/FrameworkNode.php]: you said to remove it

**Comment by ayumihamsaki** [comment]
@LukeTowers This pr does exactly the same as before - but adds a strict php logical operator.

File: modules/cms/twig/FrameworkNode.php (hunk 1 above)
Comment (LukeTowers) [reply modules/cms/twig/FrameworkNode.php]: posted that comment before the other one, sorry for the confusion

**Comment by ayumihamsaki** [comment]
This pr links to the open github issue here: https://github.com/octobercms/october/issues/4277

**Comment by LukeTowers** [comment]
Implemented in https://github.com/octobercms/october/commit/99777f5977ef50d1b8757bf0fc1e539e5e8a5afb

**Pull request closed without merging**